src/cpp/rtps/writer/ReaderProxy.cpp

```diff
@@ -645,6 +667,7 @@ ReaderProxy::ChangeConstIterator ReaderProxy::find_change(
            : it->getSequenceNumber() == seq_num ? it : end;
 }
 
+// TODO Maybe remove
```

Comment:
Please check this TODO

Reply:
I will test removing it and if all tests pass, remove it definitively.